### PR TITLE
Create sarfaraz.json

### PR DESCRIPTION
**Pull request by sarfarazstark**
To make our job easier, please spend time reviewing your application before submitting it. 

#### Add a x between [] if you meet that requirement example `[x]`
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. **This requirement can be raised if the domain you're registering is for emails**
  - Link to website: 
- [x] The website is reachable.
- [x] The CNAME record doesn't contain any slash.
- [x] There's enough information in the `owner` field.
   - You should have your email presented. If you don't want to share an email, you can leave the email an empty string (`""`) and add any other social such as Discord/Twitter/etc.
 
 #### Feel free to join our discord server for more updates or any help related to the service :D - https://discord.gg/PZCGHz4RhQ

### CODE REVIEW

**Comment by sarfarazstark** [comment]
> To make our job easier, please spend time reviewing your application before submitting it.
> 
> #### Add a x between [] if you meet that requirement example `[x]`
> * [x]  You're not using Vercel or Netlify.
> * [x]  You have completed your website, there's no type of placeholder at the website. **This requirement can be raised if the domain you're registering is for emails**
>   
>   * Link to website:
> * [x]  The website is reachable.
> * [x]  The CNAME record doesn't contain any slash.
> * [x]  There's enough information in the `owner` field.
>   
>   * You should have your email presented. If you don't want to share an email, you can leave the email an empty string (`""`) and add any other social such as Discord/Twitter/etc.
> 
> #### Feel free to join our discord server for more updates or any help related to the service :D - https://discord.gg/PZCGHz4RhQ

I reviewed my repo and I meet your given requirement.
Kindly please merge my request
Thank You

**Comment by WilliamDavidHarrison** [comment]
> I reviewed my repo and I meet your given requirement. Kindly please merge my request Thank You

Please wait, there are other PR's, not just yours.

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/sarfaraz.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "I will use this domain for my github portfolio",
+    "repo": "https://github.com/sarfarazstark/sarfarazstark.github.io",
+    "owner": {
+        "username": "sarfarazstark",
+        "email": "sarfarazstark@gmail.com",
+        "twitter": "sarfarazstark"
+    },
+    "record": {
+        "CNAME": "sarfarazstark.github.io"
```
[the remark this answers is not in the document]
Comment (sarfarazstark) [reply domains/sarfaraz.json:10]: I didn't understand what?

[the remark this answers is not in the document]
Comment (sarfarazstark) [reply domains/sarfaraz.json:10]: Ohhh you mean a web page 
I am working on it 
Please approve so I can share my website using this domain

[the remark this answers is not in the document]
Comment (sarfarazstark) [reply domains/sarfaraz.json:10]: Ok then 🙂

**Comment by sarfarazstark** [comment]
Hello please recheck and merge this commit 
i completed my portfolio

**Pull request closed without merging**